packages/react-reconciler/src/ReactFiberLane.js

```diff
@@ -555,7 +596,15 @@ export function higherPriorityLane(a: Lane, b: Lane) {
   return a !== NoLane && a < b ? a : b;
 }
 
-export function markRootUpdated(root: FiberRoot, updateLane: Lane) {
+export function createLaneMap<T>(initial: T): LaneMap<T> {
```

Comment:
Why `T` instead of a more concrete type? Do we expect to have other types of lane maps in the future?

Reply:
Most of the ones I have in mind are numbers, but not necessarily. Like the interactions map, for example.